Sources/GRPC/ConnectionBackoff.swift

```diff
@@ -38,6 +38,34 @@ public struct ConnectionBackoff: Sequence {
   /// The minimum amount of time in seconds to try connecting.
   public var minimumConnectionTimeout: TimeInterval
 
+  /// A limit on the number of times to attempt reconnection.
+  public var retries: Retries
+
+  public struct Retries {
```

Comment:
Any reason not to just make this `Hashable`?

Reply:
Done